base_dj/models/base.py

```diff
@@ -23,6 +24,13 @@ class Base(models.AbstractModel):
 
     _inherit = 'base'
 
+    @staticmethod
+    def _hash_them(atuple):
+        """Return always the same hashed string for given tuple."""
+        # TODO: get a shorter but still unique hash
+        # Maybe using `hashids` lib?
+        return hashlib.md5(str(atuple).encode()).hexdigest()
```

Comment:
@guewen here I ended up w/ this solution because using `hash()` is not granted to give you always the same result (not even on the same machine). Any thoughts? I see that you want to do something like this here :wink: https://github.com/OCA/queue/issues/57

Reply:
Yes it makes sense to do that. As it works on a tuple of strings it should be fine. 

I does that in the connector's advisory lock BTW: https://github.com/OCA/connector/blob/5ece012071004a87e60f1ff17c834ca4be407975/connector/database.py#L72 (but then I have to produce an integer in my case).

I use sha1, it has a bit more bits so I think a lesser chance of collision but I don't think that's really an issue here :)

Reply:
yeah, way too long for such case :smile: